elons-toys: improve description for first stage (#2249)
exercism/go: exercises/concept/elons-toys/.docs/instructions.md

// car is now Car{speed: 5, batteryDrain: 2, battery: 98, distance: 5}
```

> Note: If as a result of execution `car.Drive()` car's battery is below 0, you can't drive the car anymore.

Reviewer: ```suggestion
Note: You should not try to drive the car if doing so will cause the car's battery to be below 0.
```

Let me know if this suggestion clears up your confusion.

The goal of that sentence is to tell the student to not drive the car if the battery would be depleted as a result.

In that regard, I think your original suggestion makes it worse because the wording `If as a result of execution car.Drive() car's battery is below 0 ...` suggests that you should try to drive the car and only then check if the battery is less than 0, and if it is, only after that you can't drive anymore, which is not what we are trying to say.

Author: Yes, it's much better. Sorry, I was trying to clear up the same confusion but failed a bit :c
